msprobe.py

```diff
@@ -102,7 +102,12 @@ def disasmMain(disassembly, pcBase=0, microcorruptionparse=False, outfile=None,
 			pcOffset = int(disasm[4:], 16) // 2 #Words vs bytes
 			#Instruction address to output
 			xrefInsAddress = hexrep(pcBase + (pcOffset * 2) + (currentPC * 2))
-			jmpxref = xrefInsAddress + ' <' + output[currentPC + pcOffset][1] + '>'
+			#misaligned address will give an error here
```

Comment:
Begin comments with a capital. As Retr0id mentioned, this isn't misaligned but jumping into the middle of an instruction, maybe update the comment for that?